lib/views/help/how.html.erb

```diff
@@ -12,6 +12,18 @@
     who support it. Decisions about the administration of the site are taken by
     this administration team, with support from mySociety’s Chief Executive and
     <a href="<%= help_about_path(:anchor => 'who') %>">Trustees</a>.
+  </p>
+   <h2 id="information_may_be_out_of_date">
+    Information you find on the site may be out of date
+    <a href="#information_may_be_out_of_date">#</a>
+  </h2>
+  <p>
+    Some public bodies will release information that is many years out of date on the site. 
+    This can happen if the requester asks for information relating to events that happened a number of years ago. 
+    This can also happen if a document has not been updated for some time, and the requester asks for the latest version. 
+    This means that some of the information that is published on the site will already be out of date on the day it is published.
+     We never remove material from the site just because it is old or because a newer version of the material exists. 
```

Comment:
I think there is a double space at the start of line 25

 ```suggestion
    This means that some of the information that is published on the site will already be out of date on the day it is published.
    We never remove material from the site just because it is old or because a newer version of the material exists. 
```